### PR TITLE
Replace net 6 leftovers

### PR DESCRIPTION
**Pull request by Jklawreszuk**
# PR Details

## Description

As the title suggests 

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My change requires a change to the documentation.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] **I have built and run the editor to try this change out.**

### CODE REVIEW

File: sources/engine/Stride.Assets/StridePackageUpgrader.cs [hunk 1]
```diff
@@ -228,13 +228,13 @@ public override bool UpgradeBeforeAssembliesLoaded(PackageLoadParameters loadPar
                             {
                                 // In case it's a single TargetFramework, add the "s" at the end
                                 tfm.Xml.Name = "TargetFrameworks";
-                                tfm.Xml.Value = "net6.0";
+                                tfm.Xml.Value = "net8.0";
```
Comment (xen2) [inline sources/engine/Stride.Assets/StridePackageUpgrader.cs:231]: This should be kept as is for proper incremental upgrade.

File: sources/engine/Stride.Assets/StridePackageUpgrader.cs [hunk 2]
```diff
@@ -228,13 +228,13 @@ public override bool UpgradeBeforeAssembliesLoaded(PackageLoadParameters loadPar
...
                                 isProjectDirty = true;
                             }
                             // Executable
                             else if ((tfm.EvaluatedValue.StartsWith("net4", StringComparison.Ordinal) || tfm.EvaluatedValue.StartsWith("net5", StringComparison.Ordinal)) && solutionProject.Type == ProjectType.Executable)
                             {
-                                tfm.Xml.Value = solutionProject.Platform == PlatformType.Windows ? "net6.0-windows" : "net6.0";
+                                tfm.Xml.Value = solutionProject.Platform == PlatformType.Windows ? "net8.0-windows" : "net8.0";
```
Comment (xen2) [inline sources/engine/Stride.Assets/StridePackageUpgrader.cs:237]: This should be kept as is for proper incremental upgrade.

File: sources/engine/Stride.Assets/StridePackageUpgrader.cs [hunk 3]
```diff
@@ -251,15 +251,15 @@ public override bool UpgradeBeforeAssembliesLoaded(PackageLoadParameters loadPar
                         if (tfm != null)
                         {
                             // Library
-                            if (tfm.EvaluatedValue.StartsWith("net6", StringComparison.Ordinal) && solutionProject.Type == ProjectType.Library)
+                            if (tfm.EvaluatedValue.StartsWith("net8", StringComparison.Ordinal) && solutionProject.Type == ProjectType.Library)
```
Comment (xen2) [inline sources/engine/Stride.Assets/StridePackageUpgrader.cs:254]: This should be kept as is for proper incremental upgrade.

File: sources/engine/Stride.Assets/StridePackageUpgrader.cs [hunk 4]
```diff
@@ -251,15 +251,15 @@ public override bool UpgradeBeforeAssembliesLoaded(PackageLoadParameters loadPar
...
                             {
                                 // In case it's a single TargetFramework, add the "s" at the end
                                 tfm.Xml.Name = "TargetFrameworks";
                                 tfm.Xml.Value = "net8.0";
                                 isProjectDirty = true;
                             }
                             // Executable
-                            else if ((tfm.EvaluatedValue.StartsWith("net6", StringComparison.Ordinal)) && solutionProject.Type == ProjectType.Executable)
+                            else if ((tfm.EvaluatedValue.StartsWith("net8", StringComparison.Ordinal)) && solutionProject.Type == ProjectType.Executable)
```
Comment (xen2) [inline sources/engine/Stride.Assets/StridePackageUpgrader.cs:262]: This should be kept as is for proper incremental upgrade.

Comment (Jklawreszuk) [reply sources/engine/Stride.Assets/StridePackageUpgrader.cs]: Sorry, I read the code too quick 🤦. Undone

File: sources/presentation/Stride.Core.Presentation.Graph/Stride.Core.Presentation.Graph_thkfichk_wpftmp.csproj [hunk 5]
```diff
@@ -35,228 +35,228 @@
   <ItemGroup />
   <Import Project="$(StrideSdkTargets)" />
   <ItemGroup>
-    <ReferencePath Include="C:\Program Files\dotnet\packs\Microsoft.WindowsDesktop.App.Ref\6.0.0\ref\net6.0\Accessibility.dll" />
+    <ReferencePath Include="C:\Program Files\dotnet\packs\Microsoft.WindowsDesktop.App.Ref\8.0.0\ref\net8.0\Accessibility.dll" />
```
Comment (Kryptos-FR) [inline sources/presentation/Stride.Core.Presentation.Graph/Stride.Core.Presentation.Graph_thkfichk_wpftmp.csproj:38]: That file looks like a left-over that shouldn't even exist. What is it doing here?

File: sources/presentation/Stride.Core.Presentation.Graph/Stride.Core.Presentation.Graph_thkfichk_wpftmp.csproj
Comment (Kryptos-FR) [inline sources/presentation/Stride.Core.Presentation.Graph/Stride.Core.Presentation.Graph_thkfichk_wpftmp.csproj:1]: That file looks like a left-over that shouldn't even exist. What is it doing here?

Comment (Jklawreszuk) [reply sources/presentation/Stride.Core.Presentation.Graph/Stride.Core.Presentation.Graph_thkfichk_wpftmp.csproj:1]: Yeah, You're right. Deletion of the file had no effect on building or running an engine

File: sources/engine/Stride.Games/Stride.Games.csproj [hunk 6]
```diff
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup>
     <StrideRuntime>true</StrideRuntime>
-    <StrideRuntimeWindowsNet6>true</StrideRuntimeWindowsNet6>
```
Comment (Eideren) [inline sources/engine/Stride.Games/Stride.Games.csproj:4]: This variable should have a more explicit name, from what I understand this is used to force dotnet to produce an assembly specifically for windows ... ? Probably to affect preprocessors in game and input ? So, something like `StrideExplicitWindowsRuntime` maybe ?

Comment (Jklawreszuk) [reply sources/engine/Stride.Games/Stride.Games.csproj:4]: I think your suggestion about the variable name captures its meaning well 👍. Lets do that

File: sources/presentation/Stride.Core.Presentation.Graph/Stride.Core.Presentation.Graph_thkfichk_wpftmp.csproj
Comment (Kryptos-FR) [reply sources/presentation/Stride.Core.Presentation.Graph/Stride.Core.Presentation.Graph_thkfichk_wpftmp.csproj:1]: Can you delete it. Then I think we can merge.

Comment (tebjan) [reply sources/presentation/Stride.Core.Presentation.Graph/Stride.Core.Presentation.Graph_thkfichk_wpftmp.csproj:1]: Lets wait for that...

**Pull request merged**